cmd/util/util.go

```diff
@@ -60,14 +61,16 @@ func hasPort(s string) bool {
 	return strings.LastIndex(s, "]") < strings.LastIndex(s, ":")
 }
 
-// ValidateAndAddPort will take a given target address and optionally add a default port
-// onto it if a port is missing. This will also take in account optional dial timeout
-// suffix and make sure the returned value has that if it exists.
-func ValidateAndAddPort(s string, port int) string {
```

Comment:
This is part of our public API. Can we add the new function without removing the old one?

Maybe you can make this

func ValidateAndAddPort(s string, port int) string {
  return ValidateAndAddPortAndTimeout(s, port, 0)
}

Reply:
Thanks for catching this! Added it back